xml/ha_config_basics.xml

```diff
@@ -696,18 +694,20 @@
     </itemizedlist>
    </sect3>
    <sect3 xml:id="sec.ha.config.basics.resources.advanced.masters">
-    <title>Multi-state Resources</title>
+    <title>Promotable Clones (Multi-state Resources)</title>
     <para>
-     Multi-state resources are a specialization of clones. They allow the
+     Promotable clones (formerly known as multi-state resources) are a
+     specialization of clones. They allow the
      instances to be in one of two operating modes (called
      <literal>master</literal> or <literal>slave</literal>, but can mean
-     whatever you want them to mean). Multi-state resources must contain
+     whatever you want them to mean). Promotable clones must contain
      exactly one group or one regular resource.
     </para>
     <para>
-     When configuring resource monitoring or constraints, multi-state
-     resources have different requirements than simple resources. For
-     details, see &paceex;, available from <link xlink:href="http://www.clusterlabs.org/doc/"/>. Refer to
+     When configuring resource monitoring or constraints, promotable
+     clones have different requirements than simple resources. For
+     details, see &paceex;. The version for Pacemaker 1.1 is available from
```

Comment:
Hopefully there will be the docs for 2.0 soon :)

Reply:
Let's hope so.